website/content/blog/2024-12-01-a-year-in-review.md

```diff
@@ -0,0 +1,114 @@
+---
+title: A Year in Review
+date: "2024-12-01"
+excerpt: Details about the educational release of Old School Adventure
+tags: foundation, source, educational
+author:  MonoGame Foundation
+image: /images/blog/cover/monogame_foundation.png
+ogImage: /images/blog/cover/monogame_foundation.png
+---
+
+# What a year it's been!
+
+It has been a year since we announced that we received a very generous donation from Re-Logic and the formation of the MonoGame Foundation. A lot has happened.
+
+### So far in the last year we:
+
+* Now have recurring subscriptions (Patreon, Github & PayPal) of almost $2000 a month
+* Refreshed the website, with more focus on the creators and their titles \- Thanks Chris (ArisTurtle)  
+* Overhauled the documentation site and increased the content by more than 500%, with even more to come  
+* Released 3.8.2
+
+* #### Appointed to the board
+
+  * President \- Dean Ellis  
+  * Chairman \- Marko Jeremic  
+  * Secretary \- Simon Jackson  
+  * Banking \- Tom Spilman  
+  * Member \- Thomas Altenburger  
+  * Treasurer \- Dominique Louis  
+  * Please read our [Board Meeting notes](https://monogame.net/blog/meeting)  
+* Regular board meetings with full transparency, keeping the community involved in the plans and actions for MonoGame to take it into the next decade.
+
+* #### Completed Bounties
+
+  * [Package up ffmpeg and ffprobe](https://github.com/MonoGame/MonoGame/issues/8241)  
+  * [Upgrade MonoGame to use BasisUniversal for cross-platform Texture Compression](https://github.com/MonoGame/MonoGame/issues/8419)  
+  * [A shared base for public and console repos](https://github.com/MonoGame/MonoGame/issues/8242)  
+  * [Implement the Direct3D 12 / GDK/GDKX backend \#8195](https://github.com/MonoGame/MonoGame/issues/8195)
+
+* #### In Progress Bounties
+
+  * [Switch the console runtime from BRUTE to NativeAOT \#8194](https://github.com/MonoGame/MonoGame/issues/8194)  
+  * [A better 2D onboarding tutorial \#8317](https://github.com/MonoGame/MonoGame/issues/8317)  
+  * [A better 3D onboarding tutorial \#8318](https://github.com/MonoGame/MonoGame/issues/8318)
+
+* #### More bounties to come in the areas of:
```

Comment:
Some of these "future bounties" overlap with the last section.  I would just omit this section and make it part of the "next year" section.